turbo.json

```diff
@@ -56,6 +52,15 @@
         "packages/*/package.json",
         "pnpm-lock.yaml"
       ]
+    },
+    "//#format": {
+      "outputs": ["node_modules/.cache/.prettiercache"],
+      "inputs": ["**/*.ts", "**/*.tsx", "**/*.md"],
```

Comment:
why the inputs?

Reply:
Because that's the glob you pass to prettier, so it seems reasonable to skip it if it's not gonna get prettiered.

Reply:
Uhm, that's true, but actually i have no idea why we are not just running Prettier on everything. No reason to micromanage this, we can just run it on every file